tests/testthat/fixtures/README.Rmd

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Where does this `/fixtures/` convention come from?

Reply:
When there are lots of loose "testing parts" needed, I like to keep them in a folder like this. I do this elsewhere. And I thought I heard the word "fixtures" used for such things and liked it better than "test-files".